### PR TITLE
graal: generate part of the config

### PR DESCRIPTION
**Pull request by ice1000**
@imkiva

### CODE REVIEW

File: base/build.gradle.kts [hunk 1]
```diff
@@ -33,6 +33,11 @@ sourceSets.main {
 
 tasks.compileJava { dependsOn(generateVersion) }
 tasks.sourcesJar { dependsOn(generateVersion) }
+tasks.withType<org.aya.gradle.GenerateReflectionConfigTask>().configureEach {
+  extraDir = buildDir.resolve("classes/java/main/org/aya/core/serde")
+  classPrefixes = listOf("SerTerm$", "SerPat$", "SerDef$")
+  packageName = "org.aya.core.serde"
```
Comment (imkiva) [inline base/build.gradle.kts:39]: What would happen if `listOf("SerTerm", "SerPat", "SerDef")` so everything under `org.aya.core.serde` can be generated?

**Review by imkiva — APPROVED** [review]

File: base/build.gradle.kts (hunk 1 above)
Comment (ice1000) [reply base/build.gradle.kts:39]: I don't know

Comment (ice1000) [reply base/build.gradle.kts:39]: I don't understand the sentence

Comment (imkiva) [reply base/build.gradle.kts:39]: I will take this. Don't worry 😄

**Comment by imkiva** [comment]
Does this mean the generator won't work if we start a fresh build (like in CI)? Because we will not have `.class` files.

**Comment by imkiva** [comment]
bors d=ice1000

**Pull request merged**